{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -31,7 +31,7 @@ per-file-ignores =
     # star imports in `__init__.py` files are ok
     */__init__.py: F401
 
-# Enables maccabe complexity checks
```

Comment:
This was actually intended, see https://github.com/PyCQA/mccabe#plugin-for-flake8

Reply:
Sure, go ahead, thank you 🙏

Reply:
added